rclpy/rclpy/parameter_client.py

```diff
@@ -0,0 +1,310 @@
+        request.names = names
+        future = self._describe_parameters_client.call_async(request)
+        if callback:
+            future.add_done_callback(callback)
+        return future
+
+    def get_parameter_types(
+        self,
+        names: List[str],
+        callback: Optional[Callable] = None
+    ) -> Future:
+        """
+        Get parameter types given names.
+
+        The result after the returned future is complete
+        will be of type ``rcl_interfaces.srv.GetParameterTypes.Response``.
+
+        Parameter type definitions are given in rcl_interfaces.msg.ParameterType
+
+        :param names: List of parameter names to get types for.
+        :param callback: Callback function to call when the request is complete.
+        :return: ``Future`` with the result of the request.
+        """
+        request = GetParameterTypes.Request()
+        request.names = names
+        future = self._get_parameter_types_client.call_async(request)
+        if callback:
+            future.add_done_callback(callback)
+        return future
+
+    def set_parameters_atomically(
+        self,
+        parameters: Sequence[Parameter],
+        callback: Optional[Callable] = None
+    ) -> Future:
+        """
+        Set parameters atomically.
+
+        The result after the returned future is complete
+        will be of type ``rcl_interfaces.srv.SetParametersAtomically.Response``.
+
+        :param parameters: Sequence of parameters to set.
+        :param callback: Callback function to call when the request is complete.
+        :return: ``Future`` with the result of the request.
+        """
+        request = SetParametersAtomically.Request()
+        request.parameters = parameters
+        future = self._set_parameters_atomically_client.call_async(request)
+        if callback:
+            future.add_done_callback(callback)
+        return future
+
+    def delete_parameters(
+        self, names: List[str], callback: Optional[Callable] = None
+    ) -> Future:
+        """
+        Unset parameters with given names.
+
+        The result after the returned future is complete
+        will be of type ``rcl_interfaces.srv.SetParameters.Response``.
+
+        :param names: List of parameter names to unset.
+        :param callback: Callback function to call when the request is complete.
+        :return: ``Future`` with the result of the request.
+        """
+        request = SetParameters.Request()
+        request.parameters = [RclpyParameter(name=i).to_parameter_msg() for i in names]
+        future = self._set_parameter_client.call_async(request)
+        if callback:
+            future.add_done_callback(callback)
+        return future
+
+    def load_parameter_file(
+        self,
+        parameter_file: str,
+        use_wildcard: bool = False,
+    ) -> Future:
+        """
+        Load parameters from a yaml file.
+
+        Wrapper around `parse_parameter_dict` and `load_dict`
```

Comment:
I don't know if this line is still relevant (e.g. I don't know what `parser_parameter_dict` and `load_dict` are).

Reply:
Not relevant anymore, i'll take it out.